src/System.IO.Pipelines/System/IO/Pipelines/PipeReader.cs

```diff
@@ -46,18 +47,18 @@ public abstract class PipeReader
         public abstract void AdvanceTo(SequencePosition consumed, SequencePosition examined);
 
         /// <summary>
-        /// Cancel to currently pending or next call to <see cref="ReadAsync"/> if none is pending, without completing the <see cref="PipeReader"/>.
+        /// Cancel to currently pending or if none is pending next call to <see cref="ReadAsync"/>, without completing the <see cref="PipeReader"/>.
         /// </summary>
         public abstract void CancelPendingRead();
 
         /// <summary>
         /// Signal to the producer that the consumer is done reading.
         /// </summary>
-        /// <param name="exception">Optional Exception indicating a failure that's causing the pipeline to complete.</param>
+        /// <param name="exception">Optional <see cref="Exception"/> indicating a failure that's causing the pipeline to complete.</param>
         public abstract void Complete(Exception exception = null);
 
         /// <summary>
-        /// Registers callback that gets executed when writer side of pipe completes.
+        /// Registers callback that gets executed when <see cref="PipeWriter"/> side of pipe complete.
```

Comment:
`Registers a callback that gets executed when the <see cref="PipeWriter"/> side of the pipe finishes.`

Reply:
completes is here because we have Complete method and want to show connection

Reply:
Sounds good. Change to `completes` (plural):
`Registers a callback that gets executed when the <see cref="PipeWriter"/> side of the pipe completes.`